migrate-config.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/* Converts old config.inc and vocabulary.ttl configuration files, into new config.ttl */
+
+// run only in cli command line mode
+if (php_sapi_name() === "cli") {
+    /**
+     * Parse the vocabularies file, and return it in two sections, the
+     * prefixes, and the rest of the configuration.
+     * @param string $vocabulariesFile vocabularies file location
+     */
+    function parse_vocabularies_file($vocabulariesFile)
+    {
+        if (!is_file($vocabulariesFile)) {
+            echo "\nInvalid vocabularies file: $vocabulariesFile\n\n";
+            exit(1);
```

Comment:
Well, I'm removing it, but I thought it would be better to simply exit with a `1` return code. Updated with a commit replacing the exit calls by an Exception. A good outcome is that the exception includes the message that was being printed before, so replacing 2 lines by 1.

Exit code now is 255, and the error messages include the default PHP error for an uncaught exception, plus the message.